### DISCUSSION TITLE
Frequently stuck during "Preparing the installer…" phase v. 31

### DISCUSSION THREAD

**Discussion by macinthass** [grahampugh/erase-install#456] [General]
I have posted about this before and it appears to be an ongoing issue for us. Using this company-wide to keep +5K Macs uptodate with Nudge. Just tried updating 3 Macs to macOS 13.6.1 and 2/3 got stuck during the final preparation stage and errored out eventually. The progress bar goes back and forth without ever showing progress in percentage. Is there some log data I can look up and share for you to investigate why this happens? Thanks!

![Screenshot 2023-10-25 at 5 12 39 PM](https://github.com/grahampugh/erase-install/assets/77128235/f382c928-2949-47a8-ba41-93e46d34b097)

**Comment by grahampugh** [comment]
Yes there is log data in /Library/Management/erase-install/logs

**Comment by macinthass** [comment]
I am attaching four logs from a Mac that couldn’t update. Thanks!

￼￼￼￼
